src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using useMemo, but in others like here, you're not).